docs/topics/secured-booster-minishift.adoc

```diff
@@ -1,7 +1,6 @@
 = Configuring Your {OpenShiftLocal}
-Before you can run the Secured Booster, you must update your {OpenShiftLocal} from the default configuration. You only need to do this once for the secured booster missions.
+Before you can run the Secured Booster, you must update your {OpenShiftLocal} from the default configuration. You only need to do this once for the secured booster mission.
```

Comment:
Maybe "You only need to do this once for any of the secured boosters."? (Alternatively, "for any of the secured mission boosters") That would work well together with the following sentence.

Reply:
(Frankly, in situation like this, I'm starting to hate the boosters/missions/whatever metaphor we use.)

Reply:
I see what you mean. I updated based on your feedback. Thanks for the input.

Reply:
I don't see a change... :-)

Reply:
@Ladicek sorry, i had commented before i pushed, it should be in there now